content/vendor/guides/helm-chart.md

```diff
@@ -33,20 +31,20 @@ We can get the Helm Chart archive in a couple of ways, either using `helm fetch`
 
 Add a Helm repo
 
-```shell
+```text
 $ helm repo add stable https://kubernetes-charts.storage.googleapis.com
 
 "stable" has been added to your repositories
 ```
 
 Verify the repo has been added
 
-```shell
+```text
```

Comment:
since the contents aren't valid `bash` source, I have changed a few of these  to text to avoid them being parsed as shell scripts